### ISSUE TITLE
Issues with file output when the name contains `.` and a `.svg` render is requested

### ISSUE THREAD

**Issue by anishshobithps** [baraa-almasri/dsc_logo_generator#6]
As the title suggests, if the college/university name contains a `.`, such as `St. Joseph Engineering College`, the `.png` does not download due to the `St.`

![image](https://user-images.githubusercontent.com/38991749/216544718-f5401662-d222-488a-bb1d-275dee09c329.png)

It works perfectly when I remove it

![image](https://user-images.githubusercontent.com/38991749/216545030-3a207644-a9a9-457c-a76e-8ecb4a42aab0.png)

Also, it would be nice if there was a way to download render images in `.svg` format rather than `.png` format, or perhaps an option?

**Comment by mbaraa** [comment]
thanks for your notice,
if you can make a pull request with the 
suggested changes that would be great.

if not, then I'll fix them as soon as I can!

**Comment by mbaraa** [comment]
Fixed the file name issue so far,
for the SVG output, should it be any transparent SVG or like the gdsc logo template's svg?

**Comment by anishshobithps** [comment]
> Fixed the file name issue so far, for the SVG output, should it be any transparent SVG or like the gdsc logo template's svg?

Thanks for the fixing the name part, for the svg output, its better if its transparent because I have seen multiple use it as website logo

**Comment by mbaraa** [comment]
there is a transparent background with the regular PNG download
![ezgif com-gif-maker](https://user-images.githubusercontent.com/26228836/216791956-0c93679c-3c87-4a65-bb2f-d08da62971e8.gif)

**Comment by anishshobithps** [comment]
> there is a transparent background with regular PNG download

Yes, maybe I am a man with who likes clarity 💀, if its not possible no worries :) . As `SVG` is a vector it looks cleaner and does not pixelate hence the request

**Comment by mbaraa** [comment]
ok I'll see what I can do with that 👍🏼

**Comment by mbaraa** [comment]
also if you know Go or SvelteKit, feel free to open a pull request, I'm kind of busy these days!

**Comment by anishshobithps** [comment]
There is no hurry :), I don't have any knowledge in Go and I have only tried SvelteKit once, I am practically new to web development all together, all the period (`.`) issue was more important thanks for the fix 👍🏼

**Comment by mbaraa** [comment]
Okay then, wish you all the luck,
PS I noticed that you're building a GDSC website, you could use our [template](https://github.com/GDSC-ASU/website) if you like to, or even help us to make it better!
